service/agentconf.go

```diff
@@ -41,22 +41,22 @@ type SystemdServiceManager interface {
 	FindAgents(dataDir string) (string, []string, []string, error)
 
 	// WriteSystemdAgents creates systemd files and create symlinks for the
-	// list of machine and units passed in the standard filepath.
+	// machine passed in the standard filepath.
 	WriteSystemdAgents(
-		machineAgent string, unitAgents []string, dataDir, symLinkSystemdMultiUserDir string,
-	) ([]string, []string, []string, error)
+		machineAgent string, dataDir, symLinkSystemdMultiUserDir string,
+	) error
 
 	//CreateAgentConf creates the configfile for specified agent running on a
 	// host with specified series.
 	CreateAgentConf(agentName string, dataDir string) (common.Conf, error)
 
-	// CopyAgentBinary copies all the tools into the path specified for each agent.
+	// CopyAgentBinary copies the tools into the path specified for the machine agent.
 	CopyAgentBinary(
-		machineAgent string, unitAgents []string, dataDir, toSeries, fromSeries string, jujuVersion version.Number,
+		machineAgent string, dataDir, toSeries, fromSeries string, jujuVersion version.Number,
 	) error
 
 	// StartAllAgents starts all the agents in the machine with specified series.
-	StartAllAgents(machineAgent string, unitAgents []string, dataDir string) (string, []string, error)
+	StartAllAgents(machineAgent string, dataDir string) (string, error)
```

Comment:
Looks like this is never called except in tests. Might as well turf it.

Reply:
Turfing.